.github/workflows/test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Check warnings
         run: mix compile --warnings-as-errors
       - name: Run tests
-        run: mix test
+        run: mix test --exclude k8s
```

Comment:
In that case, I would always exclude it, so it would be run only when `mix test --include k8s`.